src/services/slack.ts

```diff
@@ -72,7 +67,16 @@ export class SlackConnector implements ISlackConnector {
       }
 
       values['deploy_option'] = 'deploy_all';
-      values['repo_option'] = await repoBranchesRepository.getProdDeployableRepoBranches();
+      //go through all options in dropdown by option group
+      //append version to repo_option if active
+      values['repo_option'] = [];
+      for (const group of optionGroups) {
+        values['repo_option'].push(
+          ...group.options.filter((option) => {
+            !option.text.text.startsWith('(!inactive)');
```

Comment:
We should return this to ensure things are being filtered as expected. Right now, I believe the array would be empty